boards/prusa-einsy/config.cfg

```diff
@@ -36,5 +36,5 @@ aliases:
 
 
 [mcu]
-serial: /dev/prusa-i3-mk3sp
+serial: /dev/ttyACM0
```

Comment:
/dev/prusa-i3-mk3sp doesn't work? Assuming it's flashed with firmware compiled with the config from RatOS, and you've run ratos-update.sh to symlink the udev rule, it should work.

Reply:
There was a mistake with the name of the device, in the rules file was called /dev/prusa-einsy, so finally I left that way.

Reply:
Looks good now 👍